wiki/mapping/basic-audio.md

```diff
@@ -63,10 +93,10 @@ This is to confirm that the BPM you have found online or manually matches the au
 3. Place the cursor at the start of the new track (Click on the track and press your Home key) and then click `Generate menu > Rhythm Track…`
 4. Input the nearest whole BPM you got earlier into the `Tempo (bpm)` field and enter the duration of the song in the optional `Rhythm track duration` field (the duration is displayed at the top right of the timeline).
 5. Copy the other recommended settings below:
-![Audacity Rhythm Track Menu](https://i.imgur.com/jC6OU0i.png)
+![Audacity Rhythm Track Menu](./images/clicktrack.png)
```

Comment:
This file did not load for me. Does it load for you?
![image](https://user-images.githubusercontent.com/27714637/73142846-35a43b80-4048-11ea-90ff-437f0e393101.png)

Reply:
![image](https://user-images.githubusercontent.com/27714637/73142907-c4b15380-4048-11ea-9612-666e8e4e062a.png)

Reply:
I've noticed that `click-track-1.png` and `clicktrack.png` are the same file. This loads with `click-track-1.png`.
I wonder if VuePress doesn't like that there are two nearly identical files with different names.

Reply:
Both versions of the image are building locally. I have swapped to click-track-1.png (to be consistent with later images) and deleted the problematic version... pushing a commit momentarily.
![image](https://user-images.githubusercontent.com/57695406/73143252-e2d97c80-4065-11ea-9cf0-2003c5c89192.png)